### PR TITLE
Replicant support

### PR DESCRIPTION
**Pull request by DarkmatterVale**
This PR pretty much redoes most of Tipster, giving it replicant support.

### CODE REVIEW

File: Tipster/MainWindow.cpp [hunk 1]
```diff
@@ -102,35 +102,39 @@ MainWindow::MessageReceived(BMessage* msg)
...
-			}
+			fTipsterView->UpdateTip();
+			break;
+		}
+		case PREVIOUS_TIP:
+		{
+			fTipsterView->DisplayPreviousTip();
+			break;
+		}
+		case DELAY_30_S:
```
Comment (pulkomandy) [inline Tipster/MainWindow.cpp]: Doesn't this code feel a bit repetitive?
I suggest you have a look at BMessage->AddInt32 and BMessage->FindInt32, which will allow you to use a single message constant with different values.

File: Tipster/Tipster.cpp [hunk 2]
```diff
@@ -33,14 +41,144 @@ enum
...
+	fArtworkTitle = new BString("");
+
+	fTipsterTextView = new TipsterText();
+	fIcon = new BButton("iconview", "", new BMessage(OPEN_URL));
+	fIcon->SetFlat(true);
+	
+	BRect rect(Bounds());
+	rect.top = rect.bottom - 7;
+	rect.left = rect.right - 7;
+	BDragger* dragger = new BDragger(rect, this,
```
Comment (pulkomandy) [inline Tipster/Tipster.cpp:61]: You should use the constructor without a "rect" argument, as you are in a layouted view.

Comment (DarkmatterVale) [inline Tipster/Tipster.cpp:61]: It does not display properly if I remove the rect. In fact, it does not display at all.

**Comment by humdingerb** [comment]
Guys, shouldn't ValeT's PR be merged before Hannah works on localization?

**Comment by mmuman** [comment]
Indeed.

**Comment by mmuman** [comment]
I'll just merge as is, he can always open a new one for more fixes.
I checked, #5 can be rebased with a single conflict, good occasion to learn git rebase.

**Comment by mmuman** [comment]
@DarkmatterVale please make sure you addressed all comments, if not open a new PR.

**Comment by DarkmatterVale** [comment]
@mmuman I believe I have addressed all comments.

The only comment that I am "ify" on is the one regarding the way I instantiate the BDragger. As I stated in  my answer to that comment, I am unable to view the dragger at all if I do not instantiate it the way I do.

**Pull request merged**